e2e/support/helpers/e2e-embedding-helpers.js

```diff
@@ -169,7 +169,7 @@ export function openStaticEmbeddingModal({
     cy.findByRole("button", { name: "Save" }).click();
   }
 
-  cy.findByTestId("sharing-pane-static-embed-button").click();
+  cy.findByText("Static embedding").click();
```

Comment:
nit: would it be better to use a new test ID instead of findByText?

Reply:
I usually try to stick to the approach of "The more your tests resemble the way your software is used, the more confidence they can give you.", and test ids are purely implementation details.
Any reason you think they're better in this case?

Reply:
https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change

Reply:
ah that's a classic article. I get what you mean, sounds good 😄